x-pack/test/security_functional/tests/login_selector/basic_functionality.ts

```diff
@@ -121,24 +120,25 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         .post('/authentication/app/setup')
         .send({ simulateUnauthorized: false })
         .expect(200);
-      await PageObjects.security.loginSelector.login('basic', 'basic1');
+      await PageObjects.security.loginSelector.login('basic', 'basic1', {
```

Comment:
__note:__ to solve https://github.com/elastic/kibana/issues/98562#issuecomment-833555712 the idea is to use chromeless app that doesn't make any unnecessary requests that may trigger logout earlier than we need.